app/views/pages/_portfolio.haml

```diff
@@ -0,0 +1,78 @@
+%section#portfolio
+  .container
+    .row
+      .col-lg-12.text-center
+        %h2= t('portfolio')
+        %hr.star-primary
+    .row
+      .col-sm-6.portfolio-item
+        %a.portfolio-link{"data-toggle" => "modal", :href => "#portfolioModalCoach"}
+          .caption
+            .caption-content
+              %i.fa.fa-search-plus.fa-3x
+          %img.img-responsive{:alt => "Coach", :src => image_path("portfolio/coach.png") }/
+      .col-sm-6.portfolio-item
+        %a.portfolio-link{"data-toggle" => "modal", :href => "#portfolioModalCF"}
+          .caption
+            .caption-content
+              %i.fa.fa-search-plus.fa-3x
+          %img.img-responsive{:alt => "careerfoundry", :src => image_path("portfolio/cf.png") }/
+
+    .row
+      .col-lg-12.text-center
+        = t('more_to_come_html')
+/ Portfolio Modals
+#portfolioModalCoach.portfolio-modal.modal.fade{"aria-hidden" => "true", :role => "dialog", :tabindex => "-1"}
+  .modal-content
+    .close-modal{"data-dismiss" => "modal"}
+      .lr
+        .rl
+    .container
+      .row
+        .col-lg-8.col-lg-offset-2
+          .modal-body
+            %h2 Coach by CareerBuilder
+            %hr.star-primary/
+            %img.img-responsive.img-centered{:alt => "Coach", :src => image_path("portfolio/coach.png") }/
+            %p= t('portfolio_coach_desc_html')
+            %ul.list-inline.item-details
+              %li
+                = t('portfolio_company_title')+":"
+                %strong
+                  %a{:href => "http://www.careerbuilder.com"} CareerBuilder
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace